### DISCUSSION TITLE
Example request for plotting an image in CustomTinker

### DISCUSSION THREAD

**Discussion by billalkuet07** [TomSchimansky/CustomTkinter#450] [General]
Hi I am trying to develop an image viewer using customTinker. I can take an image input by following code but do not know how to show in specific location. Here is my code.

        #opening images
        self.image1=ImageTk.PhotoImage(Image.open(os.getcwd()+"\\5FB3A2D8.JPG"))
        
```
        self.frame = customtkinter.CTkFrame(master=self.frame_right,
                               width=800,
                               height=800,
                               corner_radius=10)
        self.frame.rowconfigure(0, weight=1)
        self.frame.columnconfigure(0, weight=1)
        self.frame.grid(row=2, column=0, columnspan=3, rowspan=1, pady=20, padx=20, sticky="nsew")
        self.label_info = customtkinter.CTkLabel(master=self.frame,
                                                   image=self.image1)
```

Now, how I can show/plot the image inside the  self.frame? could you help me by small snippent?

**Comment by plugeit** [comment]
from PIL import Image,ImageTk
graphic_image=Image.open("graphic.png")
graphic_image=graphic_image.resize((34,34),Image.ANTIALIAS)
graphic_image=ImageTk.PhotoImage(graphic_image)
label=ctk.CTkLabel(frame,image=graphic_image)
label.place(x=830,y=540)
